tools/extend_entry_times.py

```diff
@@ -40,29 +40,50 @@ def per_line_entries(entries: list, file: TextIOWrapper):
 
 def extend_time_key(items):
 	for key, value in list(items.items()):
+		old_key = key
```

Comment:
Running this script on the current atlas.json in the PR appends an extra `, ` to all times. There should be no changes if the atlas.json is already extended.

Reply:
Turns out my solution may have more problems than as it seems. Have to fix it further.